tests/test_data.py

```diff
@@ -2072,13 +2092,15 @@ async def test_parameter(init_dataclasses_and_load_schema):
     updated_before = await data.Parameter.get_updated_before(datetime.datetime(2018, 7, 12, 12, 30))
     assert len(updated_before) == 0
     updated_before = await data.Parameter.get_updated_before(datetime.datetime(2018, 7, 14, 12, 30))
-    assert len(updated_before) == 1
-    assert (updated_before[0].environment, updated_before[0].name) == (parameters[2].environment, parameters[2].name)
+    assert len(updated_before) == (0 if halted else 1)
```

Comment:
By reading this test case I now realize that the name of the method (`get_updated_before`) doesn't make much sense anymore. We should give it a better name.

Reply:
ok. I would embed that info in the name of the method. But I am fine with a docstring as well.